RELEASE-NOTES.txt

```diff
@@ -1,11 +1,15 @@
 Unreleased
 ---
+
+1.100.1
+---
```

Comment:
For consistency with versions, I'm wondering if we should have an entry for `1.100.0` with the changes made in that version 🤔. I.e.

```
1.100.1
---
* [**] Fix crash when registering non-core blocks [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5982]

1.100.0
---
* [**] Add media inserter buttons to editor toolbar [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5900]
* [**] Update native BlockOutline component styles to remove blue border from blocks [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5833]
* [**] Move the undo/redo buttons to the navigation bar [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5889]
* [**] Update Editor block inserter button styles and default text input placeholder/selection styles [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5941]
* [**] Update Editor toolbar icons and colors [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5940]
* [**] Update media placeholders and block outline [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5969]
```

Reply:
I applied this suggestion via https://github.com/wordpress-mobile/gutenberg-mobile/pull/5985/commits/92c3486f902176163135eb5c9ac671fc0d43eae1.